tdrs-frontend/nginx/cloud.gov/buildpack.nginx.conf

```diff
@@ -61,9 +61,9 @@ http {
         set $ALLOWED_ORIGIN {{env "ALLOWED_ORIGIN"}};
 
         set $CSP "default-src 'self';";
-        set $CSP "${CSP}script-src 'self';";
-        set $CSP "${CSP}script-src-elem 'self';";
-        set $CSP "${CSP}script-src-attr 'self' 'unsafe-inline';";
+        set $CSP "${CSP}script-src 'self' 'unsafe-eval' 'unsafe-inline' http://{{env "KIBANA_BASE_URL"}}:5601;";
```

Comment:
fyi:  this unsafe-inline is showing up in security scans and if we need this setting, we should document why. the only documentation im aware of on this lives [here](https://github.com/raft-tech/TANF-app/blob/develop/docs/Technical-Documentation/Architecture-Decision-Record/016-zap-waiver.md), and im not sure if the justification is still accurate.

Reply:
related to the above: it stops after the endpoint page loads.

Reply:
@elipe17 I think this is worth addressing as part of #2843.